### PR TITLE
Yifan readme update

### PR DESCRIPTION
**Pull request by Yifinity**
test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # Shirley
-2023 FIRST Robotics Competition Charged Up
+
+2023 FIRST Robotics Competition Charged Up by First Robotics Team 3464 Simcity
+test2
```
Comment (3464Simcity-Personal) [inline README.md]: what is this?

**Review by 3464Simcity-Personal — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Yifinity) [reply README.md]: fixed

**Review by 3464Simcity-Personal — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,4 +1,2 @@
 # Shirley
-
-2023 FIRST Robotics Competition Charged Up by First Robotics Team 3464 Simcity
-
+Created by FIRST Robotics Team 3464 Simcity for 2023 FIRST Robotics Competition Charged Up.
```
Comment (3464Simcity-Personal) [inline README.md]: Create one more space below # Shirley to follow design rules.

Comment (3464Simcity-Personal) [inline README.md]: City should be capitolized

**Review by 3464Simcity-Personal — CHANGES_REQUESTED** [review]

**Review by 3464Simcity-Personal — APPROVED** [review]

**Pull request merged**